backends/graphs/graphs.h

```diff
@@ -124,7 +124,7 @@ class Graphs : public Inspector {
         boost::edge_attribute_t, GraphvizAttributes,
         boost::property<boost::edge_name_t, cstring, boost::property<boost::edge_index_t, int>>>;
     using graphProperties = boost::property<
-        boost::graph_name_t, cstring,
+        boost::graph_name_t, std::string,
```

Comment:
What's the impact of this switch? Is there any noticeable memory impact?

Reply:
These are transient attributes, so it does make sense to store them inside graph, not on the side. Simplifies logs here and here.